src/presentation/test/integration/registration/limited-partner-choice.test.ts

```diff
@@ -0,0 +1,64 @@
+import request from "supertest";
+import { LocalesService } from "@companieshouse/ch-node-utils";
+import * as config from "../../../../config/constants";
+import enTranslationText from "../../../../../locales/en/translations.json";
+import cyTranslationText from "../../../../../locales/cy/translations.json";
+import app from "../app";
+import {
+  LIMITED_PARTNER_CHOICE_URL,
+  NEXT_URL,
+  registrationRoutingLimitedPartnerChoice,
+} from "../../../controller/registration/Routing";
+import RegistrationPageType from "../../../controller/registration/PageType";
+
+describe("Limited Partner Choice Page", () => {
+  beforeEach(() => {
+    setLocalesEnabled(false);
+  });
+
+  const setLocalesEnabled = (bool: boolean) => {
+    jest.spyOn(config, "isLocalesEnabled").mockReturnValue(bool);
+    LocalesService.getInstance().enabled = bool;
+  };
+
+  const insertIdsInUrl = (url: string, transactionId: string, submissionId: string) => {
```

Comment:
there is already a method accessible from the dev dependencies
look at the email.test.ts